hathor/indexes/deps_index.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2021 Hathor Labs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from abc import ABC, abstractmethod
+from typing import TYPE_CHECKING, Iterator, List
+
+from hathor.transaction import BaseTransaction
+from hathor.transaction.transaction_metadata import ValidationState
+
+if TYPE_CHECKING:  # pragma: no cover
+    from hathor.transaction.storage import TransactionStorage
+
+
+INF_HEIGHT: int = 2**32 - 1
```

Comment:
Add a comment explaining that we can't increase this number directly since it would generate an error when writing it in the RocksDB index.

Reply:
OK!

Reply:
Great. Thanks!